lib/fiware-iotagent-lib.js

```diff
@@ -239,6 +224,76 @@ function deactivate(callback) {
     contextServer.stop(callback);
 }
 
+function executeIoTAgent(newConfig, iotAgent) {
+    iotAgent.start(newConfig, function (error) {
+        if (error) {
+            logger.error(context, 'Error starting JSON IoT Agent: [%s] Exiting process', error);
```

Comment:
Shit, you are right this is the node lib. You mean that the function (error) just pass the error to the up functions (toward startServer) and from there to the corresponding IoTAgent - JSON process. Is it correct?

Reply:
Yes - signature of 

```javascript
executeIoTAgent(newConfig, iotAgent , callback) {
```

Would allow the logging to be placed in each of the IoT Agents individually